packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -263,7 +263,8 @@ export const DoDontItem = React.forwardRef<HTMLDivElement, DoDontItemProps>(
                             </div>
                         )}
                         <div className="tw-w-full tw-flex tw-items-center">
-                            <h3
+                            <span
+                                className="a-h3"
```

Comment:
Can we use the theme settings here maybe to get the heading3 styles?

Reply:
They are already used, see the lines below... 

The class is just needed if the font-family is inherited